vanilla/scss/styles.scss

```diff
@@ -15,23 +15,74 @@ $grid-max-width: 92rem;
   color: $colors--light-theme--text-default;
 }
 
-//The following is a workaround as after spending a while exploring I can't work out how to escape the -- in the class name from the .rst file
-.p-notification-information {
-  @extend .p-notification--information;
+li > p {
+  margin-bottom: 0;
 }
 
-.p-notification-negative {
-  @extend .p-notification--negative;
-}
+// Reuses style from p-notification pattern
+// https://github.com/canonical-web-and-design/vanilla-framework/blob/main/scss/_patterns_notifications.scss#L21
+.admonition {
+  @extend %vf-is-bordered;
```

Comment:
We seem to have a `%vf-notification` placeholder, so technically 

```scss
.admonition {
  @extend %vf-notification;
  
  ...
}
```
should work I think.

Of course you will still need to copy any custom styles for child element, but it will at least partially reduce duplication.